### ISSUE TITLE
Uncaught TypeError: Cannot read property 'map' of null

### ISSUE THREAD

**Issue by xperiments** [weareindi/atom--var-that-color#3]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.18.0-beta0 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.4
**Thrown From**: [var-that-color](https://github.com/Ozpital/var-that-color) package 0.2.0

### Stack Trace

Uncaught TypeError: Cannot read property 'map' of null

```
At /Users/xperiments/.atom/packages/var-that-color/lib/var-that-color.js:29

TypeError: Cannot read property 'map' of null
    at Object.convert (/packages/var-that-color/lib/var-that-color.js:29:27)
    at /packages/var-that-color/lib/var-that-color.js:11:50)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app/src/command-registry.js:265:1)
    at /Applications/Atom Beta.app/Contents/Resources/app/src/command-registry.js:3:1
    at Object.didConfirmSelection (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/command-palette/lib/command-palette-view.js:106:1)
    at SelectListView.confirmSelection (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/atom-select-list/src/select-list-view.js:313:1)
    at HTMLDivElement.core:confirm (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/atom-select-list/src/select-list-view.js:81:1)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app/src/command-registry.js:265:1)
    at /Applications/Atom Beta.app/Contents/Resources/app/src/command-registry.js:3:1
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:610:1)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:401:1)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom Beta.app/Contents/Resources/app/src/window-event-handler.js:100:1)
    at HTMLDocument.<anonymous> (/Applications/Atom Beta.app/Contents/Resources/app/src/window-event-handler.js:3:1)
```

### Commands

```
     -0:31.6.0 core:move-left (input.hidden-input)
     -0:26.6.0 command-palette:toggle (input.hidden-input)
     -0:24.1.0 core:confirm (input.hidden-input)
     -0:24.1.0 var-that-color:convert (input.hidden-input)
     -0:21.3.0 editor:consolidate-selections (input.hidden-input)
     -0:21.3.0 core:cancel (input.hidden-input)
     -0:18.6.0 intentions:highlight (input.hidden-input)
     -0:17.6.0 core:select-right (input.hidden-input)
     -0:17.3.0 core:backspace (input.hidden-input)
     -0:15.6.0 intentions:highlight (input.hidden-input)
     -0:13.6.0 command-palette:toggle (input.hidden-input)
     -0:11.6.0 core:confirm (input.hidden-input)
     -0:11.6.0 var-that-color:convert (input.hidden-input)
     -0:08.7.0 command-palette:toggle (input.hidden-input)
     -0:07.4.0 core:confirm (input.hidden-input)
     -0:07.3.0 var-that-color:convert (input.hidden-input)
```

### Non-Core Packages

```
apm-publish 0.3.2 
atom-alignment 0.13.0 
atom-beautify 0.29.24 
atom-bootstrap3 1.2.12 
atom-jshint 2.0.0 
atom-scad-preview 0.6.1 
atom-typescript 11.0.3 
block-comment-plus 0.4.0 
build 0.67.0 
busy-signal 1.4.1 
double-tag 0.8.0 
drop-base64-encoding 0.2.0 
editorconfig 2.2.2 
file-icons 2.1.6 
git-blame 1.1.0 
goto-definition 1.3.3 
html-to-css 0.2.2 
intentions 1.1.2 
keysheet 0.2.1 
language-ini 1.19.0 
language-jade 0.7.2 
language-openscad 0.4.3 
language-vue 0.21.2 
linter 2.1.4 
linter-gcc 0.7.1 
linter-scss-lint 3.1.0 
linter-ui-default 1.5.4 
local-history 4.2.0 
open-in-browser 0.5.0 
open-terminal-here 2.3.0 
pigments 0.39.1 
remember-folds 0.3.0 
selection-to-tab 0.0.0 
sort-lines 0.14.0 
Stylus 3.1.0 
Sublime-Style-Column-Selection 1.7.4 
svgo 2.8.5 
sync-settings 0.8.1 
term3 0.22.1 
todo-show 1.11.0 
tool-bar 1.1.0 
unity-ui 2.1.10 
var-that-color 0.2.0 
```

**Comment by loz-archer** [comment]
Hi @xperiments - Thanks for raising this as an issue.

I'm able to reproduce the issue when nothing is selected and I execute the convert command.

Is that the same for you? Do let me know.

I'll introduce a fix asap.

**Comment by loz-archer** [comment]
This is now fixed with patch [0.2.2](https://github.com/Ozpital/var-that-color/releases/tag/v0.2.2).

Any other issues, let me know.
